modules/codegen-plugin/src/smithy4s/codegen/Smithy4sCodegenPlugin.scala

```diff
@@ -443,14 +443,16 @@ object Smithy4sCodegenPlugin extends AutoPlugin {
       ) {
         Function.untupled {
           Tracked.lastOutput[(Boolean, CodegenArgs), Seq[File]](
-            s.cacheStoreFactory.make("output")
+            s.cacheDirectory / "smithy4s-output"
           ) { case ((inputChanged, args), outputs) =>
             if (inputChanged || outputs.isEmpty) {
+              s.log.debug("Regenerating managed sources")
```

Comment:
For local debugging I also used json converter to serialize `codegenArgs` and wrote then to a tmp file for comparison. Do you think this is an useful thing to add upstream?

Reply:
I think logging the params here is OK (on `debug`)